django/contrib/admin/options.py

```diff
@@ -1728,6 +1728,22 @@ def changelist_view(self, request, extra_context=None):
             'admin/change_list.html'
         ], context)
 
+    def get_deleted_objects(self, objs, opts, request, using):
+        """
+        Hook for overriding the objects that are permitted to be deleted
+        or for customizing the format of the list of strings returned to the
```

Comment:
I'm not sure what the use case "customizing the format of the list of strings" means.  I think this would be sufficient:
```
Hook for customizing the delete process for the delete view and the
"delete selected" action.
```
In my opinion, we don't need to repeat the docstring from `get_deleted_objects()`.